api/policies/authTenant.js

```diff
@@ -104,19 +119,22 @@ module.exports = (req, res, next) => {
 
                   // be sure to set the session:
                   // req.session.tenant_id = req.ab.tenantID;
-               }
 
-               next();
+                  next();
+                  return;
+               }
+               res.notFound(`We couldn't find the tenant '${prefix}'.`);
+               return;
             }
          );
       } else {
          req.ab.log("authTenant -> no valid tenant options");
          // no Tenant ID known for this request
-         // just keep going:
-         next();
+         res.notFound(`We couldn't find a valid tenant.`);
+         return;
```

Comment:
Question: do we have a route that doesn't require a TenantID?  I can't think of one at the moment, but did we have a reason for letting the request continue without one?

Reply:
None that I'm aware of. I think we would continue so that the user could select from the drop down.